docs/usage/configuration-options.md

```diff
@@ -10,15 +10,16 @@ Any config you define applies to the whole repository (e.g. if you have a monore
 
 You can store your Renovate configuration file in one of the following locations:
 
-- `.github/renovate.json`
-- `.github/renovate.json5`
-- `.gitlab/renovate.json`
-- `.gitlab/renovate.json5`
-- `.renovaterc.json`
-- `renovate.json`
-- `renovate.json5`
-- `.renovaterc`
-- `package.json` _(within a `"renovate"` section)_
+renovate.json
+renovate.json5
+.github/renovate.json
+.github/renovate.json5
+.gitlab/renovate.json
+.gitlab/renovate.json5
+.renovaterc
+.renovaterc.json
```

Comment:
Restore the list change. For `package.json` we can add a note that it's deprecated